hadoop-crypto/src/test/java/com/palantir/crypto2/hadoop/EncryptedFileSystemTest.java

```diff
@@ -359,4 +361,61 @@ public void testDelete_fileAlreadyDeleted() throws IOException {
         assertFalse(efs.delete(path, false));
         assertThat(keyStore.get(path.toString()), is(nullValue()));
     }
+
+    @Test
+    public void testExists() throws IOException {
+        mockedEfs.exists(path);
+
+        verify(mockFs).exists(path);
+    }
+
+    @Test
+    public void testListStatus() throws IOException {
+        mockedEfs.listStatus(path);
+
```

Comment:
just remove these lines? (unless you can give me a compelling reason for having them :) )